src/utils/legacy.ts

```diff
@@ -3,4 +3,5 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-export const isLegacy = Number.parseInt(window.OC?.config.version.split('.')[0] ?? '32') < 32
+const version = window.OC?.config?.version?.split('.')[0] || '32.0.0'
```

Comment:
Semantically, it looks a bit weird that we use float value as a fallback for an integer value

Reply:
```suggestion
const version = window.OC?.config?.version?.split('.')[0] || '32'
```

Reply:
yes https://github.com/nextcloud-libraries/nextcloud-vue/pull/7314